src/index.js

```diff
@@ -37,48 +36,61 @@ const BUTTONS_ARRAY = [
     },
     {
         value: "*",
-        characters: ""
     },
     {
         value: 0,
         characters: "+"
     },
     {
         value: "#",
-        characters: ""
     },
     {
         value: "Call",
-        characters: ""
     },
     {
         value: "X",
-        characters: ""
     }
 ]
 
     ; (() => {
-
         // HEADER
         const headerElement = document.querySelector("header")
         const addNumberElement = document.querySelector("p")
         addNumberElement.innerText = "Add Number"
 
+        const handleDisplayElement = () => {
+            const displayElement = document.getElementById("display")
+            console.log(displayElement)
+        }
+
         // MAIN
+
         const mainElement = document.querySelector("main")
+        const numbersToDisplay = []
 
         BUTTONS_ARRAY.forEach((button, index) => {
+            const handleDialButtonOnClick = () => {
+                numbersToDisplay.push(buttonValueElement.innerHTML)
```

Comment:
Not really - After you press button on real device, digit is immediately visible on the screen. Therefore, it is not necessary to have `number ToDisplay`.

Instead, you could call `handleDisplayElement` where you would implement adding to view number which was pressed